include/megaapi.h

```diff
@@ -6654,13 +6653,23 @@ class MegaApiLock
      * their locks.  Only use multiple of these if you need nested locking.  The destructor
      * of the object will release the lock, so it is sufficient to delete it when finished.
      * However, when using it from a garbage collected language it may be prudent to call unlock() directly.
+     *
+     * This function must be called from the same thread that called MegaApiLock::lockOnce().
```

Comment:
The same should apply to the destruction of the object

Reply:
thanks, I'll add that comment